src/PreviewCanvas/VideoContextPreview/index.js

```diff
@@ -5,6 +5,10 @@ import Row from 'react-bootstrap/Row';
 import PropTypes from 'prop-types';
 import VideoContext from 'videocontext';
 
+const secondsToHHMMSSFormat = (seconds) => {
+  return new Date(seconds * 1000).toISOString().substr(11, 8);
+};
+
```

Comment:
Moving it out of main component as it's a very generic function. Could be moved to a util file.